### ISSUE TITLE
job pages for guests

### ISSUE THREAD

**Issue by AmandaBekhoj** [mamont77/skillspeople#129]
So, the job pages available for guests should be a little different than the ones available for advisors. 

as mentioned, we are going to get a lot of jobs in, that we get from other sites. however, we still need something to distinguish our jobs from the rest. 

so, instead of a menu in the left side of the page, i want two things under the social media buttons. 

1. search function
2. 'Populære stillinger' (popular jobs) which should be a feed with only our jobs running.

![skaermbillede 2015-01-27 kl 16 15 15](https://cloud.githubusercontent.com/assets/8081414/5920877/34ac3da6-a640-11e4-89a7-c1b6ecd37957.png)

you think this is possible? :D

**Comment by mamont77** [comment]
Of course it is possible. But if I understand correctly search form may appear after implementation of this #127 task.

**Comment by AmandaBekhoj** [comment]
Perfect. 
yes :) one thing first!

**Comment by mamont77** [comment]
This task is blocked by another task #127.

**Comment by AmandaBekhoj** [comment]
yes

**Comment by mamont77** [comment]
What do you mean by "Job Feed"? Like block the "Popular jobs"?

**Comment by AmandaBekhoj** [comment]
yes, exactly :))

**Comment by mamont77** [comment]
hmm..
Amanda,
What should I do with these blocks for guests ?
![sharepoint account manager - kobenhavn - innofactor skillspeop 02-12-2015 15 05 25](https://cloud.githubusercontent.com/assets/970967/6167900/acc6a148-b2c8-11e4-81b4-d4ab097720df.png)

**Comment by AmandaBekhoj** [comment]
hm, i think it should still be there, but maybe it could use the 'short title' ?

**Comment by mamont77** [comment]
>>but maybe it could use the 'short title' ?

Fixed.
But I have an open question still.
Should I add "Latest Jobs" block beside?

The search form in process.

**Comment by AmandaBekhoj** [comment]
which block is the 'latest jobs' ?

**Comment by mamont77** [comment]
Sorry, I wanted to ask what should be in this place, in addition to the existing blocks.
![job pages for guests issue 129 mamont77 skillspeople - goog 02-12-2015 16 19 16](https://cloud.githubusercontent.com/assets/970967/6169014/1615f540-b2d3-11e4-9483-e66cb138de65.png)

**Comment by mamont77** [comment]
And can you check the search form in mac safari?
http://www.skillspeople.dk/job/sharepoint-account-manager-koebenhavn-innofactor
![sharepoint account manager - kobenhavn - innofactor skillspeop 02-12-2015 16 21 32](https://cloud.githubusercontent.com/assets/970967/6169034/420f9b42-b2d3-11e4-8171-be1e9baa59f2.png)
Does it broken or normal?

**Comment by AmandaBekhoj** [comment]
actually i think this is all there need to be. 
and the search field works fine :)

good job!

**Comment by mamont77** [comment]
Ok. Please review and close.

Thanks a lot!

**Comment by AmandaBekhoj** [comment]
seems like all works fine! i will let you know if i find a bug ;)
